scripts/p2p-test-tool.py

```diff
@@ -282,6 +282,19 @@ def test_propose(container):
             r = container.exec_run(['sh', '-c', cmd])
             for line in r.output.decode('utf-8').splitlines():
                 print(line)
+                if "Response: Success!" in line:
+                    print(f"{container.name}: {line}")
+                    if retval != 1:
```

Comment:
This will guarantee that at least one of proposes succeeds.  This is better than it was. I'll rewrite it to work for any failure if some successes and some Failures happen on propose loops we still want it to fail.